front/src/applications/editor/tools/routeEdition/components/SearchRoute.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FiSearch } from 'react-icons/fi';
+
+import { LoaderFill } from 'common/Loaders';
+import { SearchRouteItem } from './SearchRouteItem';
+import { OptionsStateType } from '../types';
+
+interface SearchRouteProps {
+  searchFn: () => void;
+  selectFn: (i: number) => void;
+  state: OptionsStateType;
+  disabled: boolean;
+  isNew: boolean;
+}
+
+export const SearchRoute: FC<SearchRouteProps> = ({
+  searchFn,
+  selectFn,
+  state,
+  disabled,
+  isNew,
+}) => {
+  const { t } = useTranslation();
+
+  const focusedOptionIndex = useMemo(
+    () => (state.type === 'options' ? state.focusedOptionIndex : null),
+    [state]
+  );
+  return (
+    <div>
+      <button
+        className="btn btn-primary btn-sm my-1 w-100"
+        disabled={state.type === 'loading' || disabled}
+        type="button"
+        onClick={searchFn}
+      >
+        <FiSearch /> {t(`Editor.tools.routes-edition.search-routes${!isNew ? '-alt' : ''}`)}
```

Comment:
Not in the docs but the new design system will include most of the github octicons so we try to use them as much as possible when they fit our needs

```suggestion
        <GoSearch /> {t(`Editor.tools.routes-edition.search-routes${!isNew ? '-alt' : ''}`)}
```